woocommerce/woocommerce
# [Experimental] Product Filters: add individual clear button

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/46296

This PR adds a "clear filter button" block that is used by all the different filter blocks we have except for the active filters block as that one has its own.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

EXPERIMENTAL: TO BE TESTED BY DEVELOPERS ONLY

1. Go to Appearance->Editor->Templates->Product Catalog
2. Replace Products (beta) block with Products Collection block (catalog)
3. Add all the beta filters (Active, Stock, Attributes, Price, Rating).
4. Select some filters and then click on Clear for any given filter set
5. Ensure only the filter set you have clicked clear is actually cleared.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Add clear filter button block
</details>
